app/src/main/java/io/github/droidkaigi/confsched2018/presentation/staff/item/StaffItem.kt

```diff
@@ -13,7 +13,7 @@ data class StaffItem(
 ) : BindableItem<ItemStaffBinding>(staff.name.hashCode().toLong()) {
 
     override fun createViewHolder(itemView: View): ViewHolder<ItemStaffBinding> {
-        return ViewHolder(DataBindingUtil.bind(itemView))
+        return ViewHolder(DataBindingUtil.bind(itemView)!!)
```

Comment:
Probably we can use HogeHogeBinding.inflate(xxx) method. 👀 
But If you don't have time, I think that is fine as it is.